src/unity/lib/visualization/plot/plot.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef __TC_PLOT_HPP_
+#define __TC_PLOT_HPP_
+
+#include <unity/lib/toolkit_class_macros.hpp>
+#include <unity/lib/toolkit_function_macros.hpp>
+#include <unity/lib/visualization/transformation.hpp>
+#include <string>
+#include <export.hpp>
+
+namespace turi {
+  namespace visualization {
+
+    class EXPORT Plot: public toolkit_class_base{
+      public:
+
+        Plot(){};
+
+        Plot(const std::string path_to_client, const std::string vega_spec, std::shared_ptr<transformation_base> transformer, double size_array):
+                                              vega_spec(vega_spec),
+                                              path_to_client(path_to_client),
+                                              size_array(size_array),
+                                              transformer(transformer){}
+
+        void show();
+        void materialize();
+        std::string get_spec();
+
+        std::string vega_spec;
```

Comment:
Class members should be prefixed with `m_` to denote that they are members, since their usage is otherwise syntactically indistinct from scope variables. That makes the code in the `.cpp` more readable.

Reply:
changed